schematics/utils/ast.ts

```diff
@@ -60,3 +60,23 @@ export function getIndexHtmlPath(host: Tree) {
   const app = getAppFromConfig(config, '0');
   return normalize(`/${app.root}/${app.index}`);
 }
+
+/**
+ * Get the root stylesheet file.
+ */
+export function getStylesPath(host: Tree) {
+  const config = getConfig(host);
+  const app = getAppFromConfig(config, '0');
+  const styles = app.styles.find(style => {
+    const str = style.toString();
+    return str === 'styles.css' ||
+           str === 'styles.less' ||
+           str === 'styles.scss';
+  });
+
+  if (!styles) {
+    throw new SchematicsException(`Could not find global styles.ext file.`);
```

Comment:
Isn't it valid to call your root style file something else entirely? Can we read the first stylesheet from the config? Would it be better to just no-op if we can't find it?

Reply:
Well there is just a `styles` array where this lives which could include other stuff too. They could change it but I find it HIGHLY unlikely. I'll noop more gracefully.

Reply:
Yeah, I know if I was making a large app, I wouldn't want to have a file called `styles.css` because it's too vague, so I'd rename it to something like `global-styles.css` or `browser-reset.css`